### PR TITLE
Debug links not being resolved on multiple page sync

### PR DESCRIPTION
**Pull request by loudmouth**
Fix for issue #28 Where links are not resolved during multiple page sync.

Update tests to not use FRP (Interstellar) code paths.

Update to Cocoapods 1.1.1 and lock Dependencies on Swift 2.3 version

Update travis build script and Makefile

Use xcpretty for comman line builds

### CODE REVIEW

**Comment by loudmouth** [comment]
@mariobodemann notice this is a PR to merge into `Swift-2.3` not `master` ;-)

File: Sources/Entry.swift [hunk 1]
```diff
@@ -46,4 +49,61 @@ public struct Entry : Resource, LocalizedResource {
...
+            // If the passed in dictionary contains the Resource we are linking to, link it.
+            for (fieldName, fieldValue) in entryFields {
+
+                // Resolve one-to-one links.
+                if let resolvedLink = resolve(jsonFieldValue: fieldValue, againstIncludes: includes) {
+                    fields[fieldName] = resolvedLink
+                }
+
+                // Resolve one-to-many links. Some links may have already been during
+                // decoding, so we
```
Comment (mariobodemann) [inline Sources/Entry.swift]: ...

File: Sources/SignalUtils.swift [hunk 2]
```diff
@@ -16,7 +16,7 @@ func convert_signal<U, V, W>(closure: () -> (V, Signal<U>), mapper: (U) -> W?) -
         if let value = mapper($0) {
             signal.update(value)
         }
-    }.error { signal.update($0) }
+        }.error { signal.update($0) }
```
Comment (mariobodemann) [inline Sources/SignalUtils.swift]: that looks like a wierd intending.

File: Tests/ContentTypeTests.swift [hunk 3]
```diff
@@ -63,7 +63,7 @@ class ContentTypeTests: ContentfulBaseTests {
                         expect(array.items.count).to(equal(4))
 
                         let _ = array.items.first.flatMap { (type: ContentType) in
-                            expect(type.name).to(equal("Cat"))
+                            expect(type.name).to(equal("City"))
```
Comment (mariobodemann) [inline Tests/ContentTypeTests.swift]: That looks surprising: Did the cat change to a city? Or did the ordering change?

File: Tests/UserAgentTests.swift [hunk 4]
```diff
@@ -0,0 +1,24 @@
+//
```
Comment (mariobodemann) [inline Tests/UserAgentTests.swift]: :+1:

File: Sources/SignalUtils.swift (hunk 2 above)
Comment (loudmouth) [inline Sources/SignalUtils.swift]: It does look weird. Fixed :-)

**Pull request merged**